lldb-eval/eval.h

```diff
@@ -79,8 +93,11 @@ class Interpreter : Visitor {
   // types, create objects, perform casts, etc.
   lldb::SBTarget target_;
 
+  // Flow analysis contains the evaluation information about the current code
+  // branch. It allows to eliminate some "pair" operations (e.g. `&*`).
```

Comment:
Please add a more detailed description of this means - what are the elements of the vector and what null means.

Reply:
Done